project_event/models/project_project.py

```diff
@@ -40,6 +51,14 @@ def create(self, vals):
                     .next_by_code('project.project')
         return super(Project, self).create(vals)
 
+    @api.multi
```

Comment:
@jananjoy we dont need to override this method

Reply:
@jananjoy we dont need also to display the modification of the responsible_id in the chatter:responsible_id = fields.Many2one(
         'res.partner',
         string='Responsible',
         track_visibility='onchange',
     ) -> delete the track_visibility='onchange',
same thing for the title

Reply:
@rimbendhaou : Create a new task for this change request.
If we fix all issues in a single pull request, it will be hard to track or rollback if there are any issues in the main branch. At the same time, we cannot validate the task description with the pull request implemented the functionality. 

­[My Suggestion]
If you have the necessity to change existing code with the current pull request, Log or create a new task referring to impacts or issues.

Reply:
> @jananjoy we dont need to override this method

@rimbendhaou : Do you mean  that write method is not required in this context.